src/test/resources/org/jenkinsci/plugins/github/migration/MigratorTest/shouldLoadDataAfterStart/config.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <hudson>
     <disabledAdministrativeMonitors/>
-    <version>1.554.1</version>
+    <version>1.565.11</version>
```

Comment:
`matrix-plugin` was detached in `1.561` so just bump a little passed this.

unclear to me why the plugin is not failing the `InjectedTest` (which it should be as the plugin can not start due to unsatisifed dependencies...)